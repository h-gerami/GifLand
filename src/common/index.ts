export * from './image/LazyLoadImage';
export * from './searchbar/SearchBar';
export * from './loading/Loading';
export * from './button/ReloadButton';
export * from './button/CopyUrlButton';
