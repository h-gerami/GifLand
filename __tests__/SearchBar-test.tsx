import React from 'react';
import renderer from 'react-test-renderer';
import {SearchBar} from '../src/common';

test('renders correctly', () => {
  const tree = renderer
    .create(
      <SearchBar
        setSearchedText={t => console.warn(t)}
        isSearchBarFocused={false}
        searchedText={''}
        setIsSearchBarFocused={() => console.warn('')}
      />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
