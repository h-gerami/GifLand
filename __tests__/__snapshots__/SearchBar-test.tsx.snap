// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignSelf": "stretch",
      "flex": 1,
      "flexDirection": "row",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "stretch",
        "backgroundColor": "#fff",
        "borderColor": "#2A3756",
        "borderRadius": 15,
        "borderWidth": 0.5,
        "flex": 4,
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "width": 30,
        }
      }
    >
      <Text
        allowFontScaling={false}
        selectable={false}
        style={
          Array [
            Object {
              "color": "#2A3756",
              "fontSize": 25,
            },
            undefined,
            Object {
              "fontFamily": "Material Design Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        Û∞çâ
      </Text>
    </View>
    <View
      style={
        Object {
          "flex": 1,
          "justifyContent": "center",
          "paddingHorizontal": 15,
        }
      }
    >
      <TextInput
        onChangeText={[Function]}
        onFocus={[Function]}
        placeholder="Search ..."
        placeholderTextColor="#AAAAAA"
        style={
          Object {
            "color": "#2A3756",
            "fontFamily": "SofiaProMedium",
            "fontSize": 15,
          }
        }
        value=""
      />
    </View>
  </View>
</View>
`;
